packages/teuchos/core/test/MemoryManagement/CMakeLists.txt

```diff
@@ -4,15 +4,8 @@
 # executables are also likely to fail as well.
 #
 
-IF (
-  (${PACKAGE_NAME}_ENABLE_Boost AND HAVE_TEUCHOSCORE_BOOST_IS_POLYMORPHIC)
-  OR ${PROJECT_NAME}_ENABLE_CXX11
-  )
-  SET(HAS_TEUCHOS_GET_BASE_OBJ_VOID_PTR  TRUE)
-ELSE()
-  SET(HAS_TEUCHOS_GET_BASE_OBJ_VOID_PTR  FALSE)
-ENDIF()
-
+# FIXME: Remove this hard-coded setting.  We always have C++11 now.
+SET(HAS_TEUCHOS_GET_BASE_OBJ_VOID_PTR  TRUE)
```

Comment:
Here is another var that should be removed and taken out of the configured header file now that C++11 is required.